api/src/main/java/org/pmiops/workbench/api/WorkspacesController.java

```diff
@@ -598,7 +598,7 @@ public ResponseEntity<CloneWorkspaceResponse> cloneWorkspace(
           clonedRoles.put(entry.getKey(), WorkspaceAccessLevel.OWNER);
         }
       }
-      dbWorkspace = workspaceAuthService.updateWorkspaceAcls(dbWorkspace, clonedRoles);
+      dbWorkspace = workspaceAuthService.patchWorkspaceAcls(dbWorkspace, clonedRoles);
```

Comment:
here is the primary change: switch from `update` to the new `patch`

TODO: remove update entirely?